public/components/incontext_insight/generate_popover_body.test.tsx

```diff
@@ -168,7 +168,7 @@ describe('GeneratePopoverBody', () => {
         case SUMMARY_ASSISTANT_API.SUMMARIZE:
           value = {
             summary: 'Generated summary content',
-            insightAgentIdExists: false,
+            insightAgentId: undefined,
```

Comment:
Why this is undefined and not `insightAgentId`

Reply:
`insightAgentId: undefined`  means no insight agentId for this alert, which is semantic equal to `insightAgentIdExists: false`

Reply:
Makes sense